### ISSUE TITLE
Create sync management command to pull breaches and logos into db

### ISSUE THREAD

**Issue by robhudson** [mozilla/bedrock#13612]
## Description

Sub-task of issue #13553
Depends on #13611 

The breach data comes from the [JSON API endpoint at HIBP](https://haveibeenpwned.com/api/v3/breaches).
The breach icons are fetched from DDG, the current code for which is here: https://github.com/mozilla/blurts-server/blob/1382f301f0b8c6eb34b5010ce7526cf7c87dcc8a/src/scripts/syncBreaches.js#L36-L75

The management command should load breaches into the data model and fetch the logos and store them in the static assets bucket. See the [existing code](https://github.com/mozilla/blurts-server/blob/1382f301f0b8c6eb34b5010ce7526cf7c87dcc8a/src/scripts/syncBreaches.js) for guidance.

**Comment by janbrasna** [comment]
This seems like `wontfix` as the parent issues along with PRs were closed as "not planned".
